### ISSUE TITLE
Troubleshoot wave peak accuracy & peak/valley size

### ISSUE THREAD

**Issue by helencarey** [ByunLab/staRt-ionic#344]
TM: As I described in a message to Sam last week, though: "while the wave is now nicely bounded by the posts in the graphic display, it is tracking peaks with much lower accuracy than it used to. It seems like there may be some sort of range issue--the visible formants seem to cluster in the lowest (left-hand) frequencies. But there is also a general issue with peaks and valleys being neither as well-defined nor as stable as we are used to seeing."

**Comment by helencarey** [comment]
@tarabyun 
Yes, we can definitely get better defined peaks and valleys by increasing the range max, which in our case is just the top edge of our graph. (Since our design is responsive, the graph grows and shrinks to fit whatever screen area we give it. Perhaps another element, like the word prompt, is squashing it. I need to investigate this.) To fix this we could give it a fixed aspect ratio.

TM: _the visible formants seem to cluster in the lowest (left-hand) frequencies_
Is it like the left (min) and right (max) of the Hz-axis are reversed?
Or is it like they are just mapped incorrectly -- ex a peak that should be at 1000 shows up at 2200?

This is what I am seeing for /r/ in simulator. 
The peaks could be better defined. Are they in the right place for a domain of 0-4500?
![Simulator Screen Shot - iPad Air 2 - 2019-05-03 at 11 41 54](https://user-images.githubusercontent.com/8650944/57149901-ce987580-6d9a-11e9-8b30-1082590d2458.png)

**Comment by tarabyun** [comment]
Hi Helen, I answered this query in Slack but wanted to transfer my responses here for posterity.

HC: This is what I am seeing for /r/ in simulator. The peaks could be better defined. Are they in the right place for a domain of 0-4500? 
Response: This is odd; the third peak is actually too high for a typical /r/. Unless the first three peaks are actually squashed together in the low frequencies and what we're looking at is actually the 4th peak.

For comparison, the images of good/bad /r/ from your tutorial :) 

![image](https://user-images.githubusercontent.com/16907239/57541752-bf2aa680-731d-11e9-8b56-874304b7bdcf.png)

**Comment by tarabyun** [comment]
Bottom line, I guess I can't say the peaks are consistently too high or too low in frequency--they just aren't showing up where I expect them to be :shrug: although with the higher sampling rate, it's hard to confirm the locations of the peaks in the first place.

**Comment by helencarey** [comment]
**QA Checkpoints for @erinmdoty**

changes affecting: all instances of the wave (tutorial, free, quest, profiles/settings)
- [ ] wave speed is acceptable
- [ ] wave shape is correct for target sounds
- [ ] peaks and peak-lines are in proper locations

changes affecting: Free Play & Quest-BF
- [ ] star slider works as expected — responds to touch, doesn’t lag, etc.
- [ ] fz signpost should always show a the value of the slider position on the 0-4500 hz wave graph.
- [ ] the *moving* wave should only appear between the 2 posts


**vQA** 
The 2.0.6 should look this vid: 
https://drive.google.com/file/d/1LTAjlXrLNaJfVUYXavoR1qN5W-4swfYx/view?usp=sharing

**Comment by tarabyun** [comment]
@helencarey, any chance this could be assigned to Twylah instead of Erin for testing? Erin is sick, unfortunately :(

**Comment by Tjc10** [comment]
Hi all, I was able to test version 2.0.7. I tried it out as formal tester and used the profile "QACheckpoints TCTesting" to do it.  I went through baseline and then 5 sessions (2 TRAD sessions + 3 BF sessions).  I attempted to upload after each but the "Upload Error" message popped up each time. I do see the sessions in my recordings log under my profile though.  

In terms of the specific questions Helen had for Erin:
- wave speed is acceptable
- wave shape is correct for target sounds
- peaks and peak-lines are in proper locations
- star slider works as expected — responds to touch, doesn’t lag, etc.
- the moving wave should only appear between the 2 posts
YES / YES/ YES / YES / YES  on all these. NO ISSUES to report here.  All seem to be working just fine.

One thing to note: 
- fz signpost should always show a the value of the slider position on the 0-4500 hz wave graph.
YES the values do appear on the sign post.  However, my value scale only goes from 0-4017Hz. The bar does not seem to go all the way up to 4500Hz. This may not matter but just an FYI that the value I see stops at 4017Hz. Thanks.
